lib/ansible/modules/packaging/os/pkg5.py

```diff
@@ -144,16 +126,7 @@ def ensure(module, state, packages, params):
 
     to_modify = filter(behaviour[state]['filter'], packages)
     if to_modify:
-        rc, out, err = module.run_command(
-            [
-                'pkg', behaviour[state]['subcommand']
-            ]
-            + dry_run
-            + accept_licenses
-            + [
-                '-q', '--'
-            ] + to_modify
-        )
+        rc, out, err = module.run_command(['pkg', behaviour[state]['subcommand']] + dry_run + accept_licenses + ['-q', '--'] + to_modify)
```

Comment:
I find these kinds of long line less readable, harder to work with, and less helpful with `git blame`, but I realise that this is all subjective, so I'm going to try to resist arguing about the colour of this particular bikeshed.

Reply:
I don't mind doing it differently though. The original was not PEP8 compliant, but there is a PEP8 compliant multi-line solution. Our PEP8 implementation also supported 160 wide lines, so I opted for that.

Reply:
It's already merged, feel free to implement your preference